src/state-summary/state-card-analog_output.ts

```diff
@@ -11,7 +11,7 @@ import "../components/ha-slider";
 
 class StateCardAnalogOutput extends mixinBehaviors(
   [IronResizableBehavior],
-  PolymerElement
+  LitElement
```

Comment:
It isn't this easy, the API of a LitElement is different than that of a PolymerElement.

Reply:
Yep, took me some more time :)